js/map_utils.js

```diff
@@ -264,9 +285,38 @@ function showToolTip(d, category) {
   }, toolTipDelay);
 }
 
-function hideToolTip() {
-  d3.select(".tooltip")
-    .classed("hidden", true);
+function clearLegendText() {
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#legend-title")
+    .text("Water Use");
```

Comment:
I believe this was "Water use" with lowercase u the first time you set it, up in buttons.js line 22. What if rather than setting it at all up there, you simply call `clearLegendText()` from within `addButtons`?